src/components/login/login.tsx

```diff
@@ -0,0 +1,111 @@
+import { Button, Dialog, DialogActions, DialogContent, DialogContentText, DialogTitle, TextField } from '@material-ui/core';
+import * as React from 'react';
+import * as rest from "../../rest/interface";
+import '../../styles.css';
+
+interface State {
+  APIClient: rest.Evergreen;
```

Comment:
This should be a part of Props because it's data that should be set by the main app and passed to all its child components

Reply:
Fixed